app/routes/meetings/components/MeetingEditor.js

```diff
@@ -165,6 +170,12 @@ function MeetingEditor({
           </div>
         )}
 
+        <button
```

Comment:
What does this button do?

Reply:
Ops. Found it here: https://stackoverflow.com/a/51507806 (because now enter in a field submits the form and saves the meeting)

Maybe clicking enter inside the textArea works without it. Need to test.

Reply:
Aha, ok!